.github/workflows/build_and_deploy.yml

```
@@ -0,0 +1,89 @@
+    outputs:
+      commit_title: ${{ steps.get_head_commit_title.outputs.title }}
+      version: ${{ steps.generate_version.outputs.version }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - id: get_head_commit_title
+        run: echo "title=$(git log --format=%B -n 1 HEAD | head -n 1)" >> $GITHUB_OUTPUT
+
+      - id: generate_version
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:4:30: Double quote to prevent globbing and word splitting

(shellcheck)